modules/enabling-anonymous-pulls-new-cluster.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * registry/accessing-the-registry.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="enabling-anonymous-pulls-new-cluster_{context}"]
+= Anonymous pulls for new clusters
+
+Use the following procedure to enable anonymous pulls if you have installed a new cluster from {product-title} 4.16 or later.
```

Comment:
This procedure will not enable anonymous pulls to the registry. It only enables anonymous requests to the self access reviewer resource, which is required in new clusters in addition to the below step:
```
oc adm policy add-role-to-group system:image-puller system:unauthenticated --namespace <namespace>
```